rclcpp/src/rclcpp/timer.cpp

```diff
@@ -75,6 +75,17 @@ TimerBase::cancel()
   }
 }
 
+bool
+TimerBase::is_cancelled()
+{
+  bool is_canceled = false;
+  if (rcl_timer_is_canceled(timer_handle_.get(), &is_canceled) != RCL_RET_OK) {
+    throw std::runtime_error(std::string("Couldn't get timer cancelled state: ") +
+            rcl_get_error_string().str);
```

Comment:
Please use `rclcpp::exceptions::throw_from_rcl_error()` instead:

https://github.com/ros2/rclcpp/blob/713dd0c18411623ea82af1255f583a00c3c5970b/rclcpp/include/rclcpp/exceptions.hpp#L100-L119

Reply:
(and update the documentation to reflect that it no longer exclusively throws `runtime_error`)

Reply:
That's handy! Addressed in the latest commit.